main/main.cpp

```diff
@@ -1191,6 +1200,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	OS::get_singleton()->set_low_processor_usage_mode_sleep_usec(GLOBAL_DEF("application/run/low_processor_mode_sleep_usec", 6900)); // Roughly 144 FPS
 	ProjectSettings::get_singleton()->set_custom_property_info("application/run/low_processor_mode_sleep_usec", PropertyInfo(Variant::INT, "application/run/low_processor_mode_sleep_usec", PROPERTY_HINT_RANGE, "0,33200,1,or_greater")); // No negative numbers
 
+	GLOBAL_DEF("application/run/delta_smoothing", false);
```

Comment:
Shouldn't we make it enabled by default? Or said otherwise:
- Should we make it default to true during 3.4 betas to get wider testing?
- If testing is a success, do we also want it defaulted to true in 3.4 stable?

Reply:
Yes, we can certainly default to true to get better testing. I'm always wary about these things but you are right. Once we have any kinks ironed out we can potentially default to true in 3.4 stable. I'll change it to true.

Reply:
Fixed.